templates/admin_assets/stylesheets/admin.sass

```diff
@@ -0,0 +1,5 @@
+@import common/**/*
+// @import admin/**/*
+/*contains system-shared css variables*/
+@import frontend/common/bootstrap-override
```

Comment:
neměl by tenhle soubor být v `stylesheets/common`, když je to ve skutečnosti common, ale načítá se to do `admin` ze složky `frontend`?